optimade/server/routers/info.py

```diff
@@ -0,0 +1,83 @@
+from typing import Union
+
+from fastapi import APIRouter
+from starlette.exceptions import HTTPException as StarletteHTTPException
+from starlette.requests import Request
+
+from optimade.models import (
+    ErrorResponse,
+    InfoResponse,
+    EntryInfoResponse,
+    ReferenceResource,
+    StructureResource,
+)
+
+from .utils import meta_values, retrieve_queryable_properties
+
+
+router = APIRouter()
+
+ENTRY_INFO_SCHEMAS = {
+    "structures": StructureResource.schema,
+    "references": ReferenceResource.schema,
+}
+
+
+@router.get(
+    "/info",
+    response_model=Union[InfoResponse, ErrorResponse],
+    response_model_skip_defaults=False,
+    tags=["Info"],
+)
+def get_info(request: Request):
+    from optimade.models import BaseInfoResource, BaseInfoAttributes
+
+    return InfoResponse(
+        meta=meta_values(str(request.url), 1, 1, more_data_available=False),
+        data=BaseInfoResource(
+            attributes=BaseInfoAttributes(
+                api_version="v0.10",
+                available_api_versions=[
+                    {
+                        "url": "http://localhost:5000/optimade/v0.10.0/",
```

Comment:
Can we now use CONFIG.index_base_url to figure out what this url should be?

Reply:
Not a bad idea! :)

Reply:
Note: This should be put here, but rather, it should be used in the top-level `meta` response under `provider` - which it is automatically :)

What I have done in my own implementation, however, is to add `version` to the config file. In this way it can be used here and retrieved whenever needed, without having to import the `app` to get the version from there.